api/cpp/platform.rs

```diff
@@ -964,6 +964,7 @@ pub mod skia {
 
         let boxed_renderer: Box<SkiaRenderer> = Box::new(
             SkiaRenderer::new(
+                &i_slint_renderer_skia::SkiaSharedContext::default(),
```

Comment:
so no sharing with C++ and android?  But I guess we just start somewhere.

Reply:
I see android is single window atm.